front/src/reducers/editor/index.ts

```diff
@@ -64,13 +73,9 @@ export const editorSlice = createSlice({
     },
   },
 });
-
-export const {
-  selectLayers,
-  loadDataModelAction,
-  updateTotalsIssueAction,
-  updateFiltersIssueAction,
-} = editorSlice.actions;
+export const editorSliceActions = editorSlice.actions;
+const { loadDataModelAction, updateTotalsIssueAction, updateFiltersIssueAction } =
+  editorSliceActions;
 
 export type editorSliceActionsType = typeof editorSlice.actions;
```

Comment:
Types start by a Maj, so EditorSliceAction (type is maybe superflu)

Reply:
Done